app/src/main/java/app/inf1d_team3/open_day/OpenDaysActivity.java

```diff
@@ -46,6 +48,15 @@ public void onClick(View v) {
             }
         });
         recyclerView.setAdapter(openDaysAdapter);
+
+        // the third button(ask question)
+        button = (Button) findViewById(R.id.button3);
+        button.setOnClickListener(new View.OnClickListener() {
```

Comment:
Change onclick to be connect via xml instead of via the code (better compatability), look at openInfopage function & button